tests/test_check_cloudformation.py

```diff
@@ -91,5 +91,5 @@ def test_outputs_failed_results_summary(patched_prepare_cloudformation_templates
     result = CliRunner().invoke(check_cloudformation_command)
 
     assert (
-        "The CloudFormation templates failed the following checks :-(\n  - lint" in result.output
+       "The CloudFormation templates passed the following checks :-)\n  - lint\n" in result.output
```

Comment:
Hmmm... is this meant to pass? Looks like this test is supposed to output the failure of the CloudFormation templates.

Reply:
Interesting... was this test passing AKA failing the checks before because of W2001 and now it goes through because we added this check to the `--ignore-checks` flag?

Reply:
How is this resolved? The test name still says "failed", but the assert says "passed".

Reply:
@WillGibson Not sure if you are looking at the right commit, but the assert also says "failed"...

Reply:
The test method name is `test_outputs_failed_results_summary`.

Reply:
Oh, ignore. I think I just discovered that if you navigate to a file from one of these threads on the conversation tab it doesn't show you all commits by default. I wonder how much nonsense I've said because of that. Will approve again.